lib/declarations.ts

```diff
@@ -57,10 +57,16 @@ interface IOpener {
 }
 
 interface ILiveSyncService {
-	liveSync(platform: string): IFuture<void>;
+	liveSync(platform: string, applicationReloadAction?: (deviceAppData: Mobile.IDeviceAppData, localToDevicePaths: Mobile.ILocalToDevicePathData[]) => IFuture<void>): IFuture<void>;
 	forceExecuteFullSync: boolean;
 }
 
+interface IPlatformLiveSyncService {
+	fullSync(postInstallAction?: (deviceAppData: Mobile.IDeviceAppData, localToDevicePaths: Mobile.ILocalToDevicePathData[]) => IFuture<void>): IFuture<void>;
```

Comment:
the action is used even when app is already installed, so the name is not really correct. `postAction` seems better IMO, but it's just a suggestion, not a merge stopper :)